mint-005.md

```diff
@@ -0,0 +1,165 @@
+# mint-005
+
+## 3 Key Joint Custody
+
+## Motivation
+
+To provide a custody arrangement in which an owner of Bitcoin (principle) is able to secure bitcoin by working with an Agent. Unlike existing collaborative custody models up to this point, **where bitcoin keys WITHIN a multisig threshold are shared between principles and agents**, a joint custody model by default requires a threshold of keys by both the Agent & Principle for movement of funds.
```

Comment:
In 7bebe7efd503756903ccb9efc65e1e681a5ebb4e:

`principle` should be `principal` throughout, and (I think) capitalized.

Capitalization of `bitcoin` should be consistent (I think lowercase).

`Primary Agent` should be consistently capitalized throughout.

Reply:
Updated